SIGSEGV: metrics.CollectJobMetric

Trying current main (d344396706f897786cb05f25ce7aae27fbef0973), updating from 0.1.7, starting garm results in

```
# /usr/local/bin/garm -config /etc/garm/config.toml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd070ba]

goroutine 1 [running]:
github.com/cloudbase/garm/runner/metrics.CollectJobMetric({0x147fdd0, 0xc0016c4ed0}, 0xc000e8a008)
	/opt/garm/runner/metrics/jobs.go:43 +0x11a
github.com/cloudbase/garm/runner/metrics.collectMetrics({0x147fdd0, 0xc0016c4ed0}, 0xc000e8a008, {{0x73, 0x6, 0x12, 0xd7, 0x3c, 0x4, 0x4f, ...}, ...})
	/opt/garm/runner/metrics/metrics.go:105 +0x345
github.com/cloudbase/garm/runner/metrics.CollectObjectMetric({0x147fdd0?, 0xc0007be570?}, 0xc000e8a008, 0xdf8475800)
	/opt/garm/runner/metrics/metrics.go:39 +0x2f0
main.main()
	/opt/garm/cmd/garm/main.go:341 +0x1533
```

--- a/runner/metrics/jobs.go
+++ b/runner/metrics/jobs.go
@@ -40,7 +40,7 @@ func CollectJobMetric(ctx context.Context, r *runner.Runner) error {
 			job.Status,                    // label: status
 			job.Conclusion,                // label: conclusion
 			job.RunnerName,                // label: runner_name
-			job.RepoID.String(),           // label: repository
+			job.RepositoryName,            // label: repository
 			strings.Join(job.Labels, " "), // label: requested_labels
 		).Set(1)
 	}

